lib/chord.js

```diff
@@ -67,15 +71,43 @@ class Chord {
   transpose(amount) {
     return new Chord(transpose(this.root, amount), this.quality, transpose(this.bass, amount));
   }
+}
 
-  static isChord(str) {
-    return str.match(chordPattern) ? true : false;
+const compareNotes = (note1 = '', note2 = '') => NOTE_SEQUENCE.get(note1) - NOTE_SEQUENCE.get(note2);
```

Comment:
Why do we need defaults in this? Is there ever a case where this should be called with on parameter?

Reply:
Yes, you're right. The original logic here was that Chords have two notes, a root note and a bass note. The bass note is optional, but the way I wrote the Chord parser, the bass note will never be undefined, so removing the `''` does in fact work.